src/main/java/com/cinema/config/validations/EmailConstraint.java

```diff
@@ -0,0 +1,23 @@
+package com.cinema.config.validations;
```

Comment:
```suggestion
package com.cinema.validation;
```
same for both your annotations and validators